### DISCUSSION TITLE
File format discussion

### DISCUSSION THREAD

**Discussion by bsantosh909** [Know-Nepal/borderlines#1] [Polls]
It is absolutely necessary for us to understand which file format is best suited for everyone's use case.

Thus please share of the feedbacks you have on this topic.
We are all ears 😉

**Comment by diwash007** [comment]
JSON would be perfect.
If CSV or XML or any other file types are required by any parties, they can use converters to obtain the required file formats.

**Reply by ofcoursenp** [reply]
Yes Json is better for these stuffs

**Comment by casualsnek** [comment]
I would prefer JSON if data is collected/generated from automated sources, for data that needs to be managed manually YAML is good as it is human readable and supports comments unlike JSON !

**Comment by twlite** [comment]
TOML would be much more readable than YAML though. We could also setup workflows and generate other main file formats if necessary.

**Reply by casualsnek** [reply]
IMO, TOML seems be be better designed for smaller configuration files than hierarchical data storage

**Reply by twlite** [reply]
Wouldn't yaml serve similar purpose too? I was talking about readability there.

For static data storage, I would say JSON.

**Comment by thenishantsapkota** [comment]
I would suggest using JSON for static data storage.
Although all the file formats are good, json will be easier to implement and use in the long run!

**Comment by bsantosh909** [comment]
Alright JSON it is then.
And if necessary we can use the GI workflows or any other tools to convert the data to different file formats. 🙏

**Reply by twlite** [reply]
🤞Yeah let's go with this

**Reply by thenishantsapkota** [reply]
Let's do JSON!
